pkgs/applications/misc/megasync/default.nix

```diff
@@ -25,6 +25,7 @@
 , swig
 , unzip
 , wget
+, enableExtNautilus ? true, gnome
```

Comment:
```suggestion
, enableExtNautilus ? true
, gnome
```

Reply:
This dep is related to the option and it's better(?) to have it on one line